sdk/storage/Azure.Storage.DataMovement.Blobs/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.0.1 (Unreleased)
```

Comment:
interesting. I thought the bot was supposed to add this... I have to check something